packages/bsky/tests/views/quotes.test.ts

```diff
@@ -0,0 +1,105 @@
+import { quotesSeed, SeedClient, TestNetwork } from '@atproto/dev-env'
+import AtpAgent from '@atproto/api'
+import { forSnapshot } from '../_util'
+import { ids } from '../../src/lexicon/lexicons'
+
+describe('pds quote views', () => {
+  let network: TestNetwork
+  let agent: AtpAgent
+  let sc: SeedClient
+
+  // account dids, for convenience
+  let alice: string
+  let bob: string
+  let eve: string
+
+  beforeAll(async () => {
+    network = await TestNetwork.create({
+      dbPostgresSchema: 'bsky_views_quotes',
+    })
+    agent = network.bsky.getClient()
+    sc = network.getSeedClient()
+    await quotesSeed(sc)
+    await network.processAll()
+    alice = sc.dids.alice
+    bob = sc.dids.bob
+    eve = sc.dids.eve
+  })
+
+  afterAll(async () => {
+    await network.close()
+  })
+
+  it('fetches post quotes', async () => {
+    const alicePostQuotes = await agent.api.app.bsky.feed.getQuotes(
+      { uri: sc.posts[alice][0].ref.uriStr, limit: 30 },
+      { headers: await network.serviceHeaders(eve, ids.AppBskyFeedGetQuotes) },
+    )
+
+    expect(alicePostQuotes.data.posts.length).toBe(2)
+    expect(forSnapshot(alicePostQuotes.data)).toMatchSnapshot()
+  })
+
+  it('utilizes limit parameter and cursor', async () => {
+    const alicePostQuotes1 = await agent.api.app.bsky.feed.getQuotes(
+      { uri: sc.posts[alice][1].ref.uriStr, limit: 30 },
+      { headers: await network.serviceHeaders(eve, ids.AppBskyFeedGetQuotes) },
+    )
+
+    expect(alicePostQuotes1.data.posts.length).toBe(30)
+    expect(alicePostQuotes1.data.cursor).toBeDefined()
+
+    const alicePostQuotes2 = await agent.api.app.bsky.feed.getQuotes(
+      {
+        uri: sc.posts[alice][1].ref.uriStr,
+        limit: 30,
+        cursor: alicePostQuotes1.data.cursor,
+      },
+      { headers: await network.serviceHeaders(eve, ids.AppBskyFeedGetQuotes) },
+    )
+
+    expect(alicePostQuotes2.data.posts.length).toBe(20)
+  })
+
+  it('does not return post when quote is deleted', async () => {
+    await sc.deletePost(eve, sc.posts[eve][0].ref.uri)
+    await network.processAll(1000)
+
+    const alicePostQuotes = await agent.api.app.bsky.feed.getQuotes(
+      { uri: sc.posts[alice][0].ref.uriStr, limit: 30 },
+      {
+        headers: await network.serviceHeaders(alice, ids.AppBskyFeedGetQuotes),
+      },
+    )
+
+    expect(alicePostQuotes.data.posts.length).toBe(1)
+    expect(forSnapshot(alicePostQuotes.data)).toMatchSnapshot()
+  })
+
+  it('does not return any quotes when the quoted post is deleted', async () => {
+    await sc.deletePost(alice, sc.posts[alice][0].ref.uri)
+    await network.processAll(1000)
```

Comment:
Is the timeout required, or leftover from debugging?  If there's a known reason it's required may just be nice to leave a comment.

Reply:
probs copy/paste 👍